blockchain/My-Wallet-V3-iOS
# update(SocketMessage): IOS-1417 use new parameter names

## Objective

Update socket message models.

## Description

- `operation` renamed to `action`
- `type` renamed to `event`
- `sequenceNumber` renamed to `seqnum`

## How to Test

Go to Exchange when changes have been deployed, rates should update as expected.

## Screenshot/Design assessment

N/A

## Merge Checklist
